planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -808,6 +820,41 @@ void planning_scene_monitor::PlanningSceneMonitor::currentWorldObjectUpdateCallb
     }
 }
 
+void planning_scene_monitor::PlanningSceneMonitor::syncSceneUpdates(const ros::Time &t)
+{
+  if (t.isZero())
+    return;
+
+  ROS_DEBUG_STREAM_NAMED("PSM", "sync to: " << fmod(t.toSec(), 10.));
+  enforce_next_state_update_ = true;  // enforce next state update to trigger without throttling
```

Comment:
Nice catch! Actually the variable isn't really necessary anymore with the latest patches.
It was meant to quickly forward a state update to `syncUpdates()` to minimize waiting.
But, yes, indeed the variable should be reset to true in syncUpdates's while loop to keep that effect. However, I suggest to remove it.